packages/kit/rollup.config.js

```diff
@@ -23,6 +23,10 @@ export default [
 			'app/stores': 'src/runtime/app/stores.js',
 			'app/paths': 'src/runtime/app/paths.js',
 			'app/env': 'src/runtime/app/env.js',
+			'env/dynamic/private': 'src/runtime/env/dynamic/private.js',
+			'env/dynamic/public': 'src/runtime/env/dynamic/public.js',
+			'env-private': 'src/runtime/env-private.js',
```

Comment:
felt unnecessary since we don't have entry points for `static`. in any case i'm thinking of these as temporary — this whole folder structure is due for a reshuffle (it's weird that we have `env` alongside `env-[type]`, for example)